src/screens/Document/index.js

```diff
@@ -37,6 +37,7 @@ class DocumentScreen extends Component {
             docType: 'Document Type',
             notes: ''
         },
+        docStatus: 'UPLOAD',
```

Comment:
Maybe have the wording be 'UPLOAD_NEW' for new documents, and 'SAVE_EDITS' for existing documents (or something like that to make it more clear which one is for new documents and which is for existing documents.